web/app.js

```diff
@@ -1009,12 +1009,26 @@ const PDFViewerApplication = {
       return;
     }
 
+    if (AppOptions.get("enableScripting") && this._scriptingInstance) {
```

Comment:
Here, and elsewhere, it should suffice to check `if (this._scriptingInstance) {` since that cannot happen without the "enableScripting"-option being set.